demisto_sdk/commands/common/errors.py

```diff
@@ -482,6 +483,12 @@ def readme_missing_output_context(command, context_paths):
         return f'The Following context paths for command {command} are found in YML file ' \
                f'but are missing from the README file: {context_paths}'
 
+    @staticmethod
+    @error_code_decorator
+    def error_starting_mdx_server(line, packs):
+        return f'Failed starting mdx server. stdout: {line}.\n' \
+               f'Try running the following command: `npm install {" ".join(packs)}`'
```

Comment:
The command to run is simply: `npm install`. There is no need to specify the packages as they are specified in `package.json`.
```suggestion
               f'Try running the following command: npm install'
```

Reply:
done